src/utilities/accessibility-helper.ts

```diff
@@ -0,0 +1,14 @@
+export function accessibilityClick(event: any) {
+  if (event.type === "click") {
+    return true;
+  }
+  else if (event.type === "keydown") {
+    const code = event.charCode || event.keyCode;
+    if ((code === 32) || (code === 13)) {
+      return true;
+    }
+  }
+  else {
+    return false;
+  }
+}
```

Comment:
Add newline at end of file and consider naming module `accessibility-utils.ts` to match others.